### DISCUSSION TITLE
Error: Failed to load config "@ljharb" to extend from.

### DISCUSSION THREAD

**Discussion by oimmc** [eslint/eslint#14544] [Q&A]
**Tell us about your environment**

* **ESLint Version:*v7.22.0*
* **Node Version:*v12.21.0*
* **npm Version:*6.14.11*
* **Operating System:*macOS Big Sur v11.2.3* 

**.eslintrc.js**
```
module.exports = {
	env: {
		browser: true,
		es6: true,
		node: true
	},
	extends: ['eslint:recommended', 'plugin:react/recommended'],
	globals: {
		Atomics: 'readonly',
		SharedArrayBuffer: 'readonly'
	},
	parserOptions: {
		ecmaFeatures: {
			jsx: true
		},
		ecmaVersion: 2018,
		sourceType: 'module',
		project: ['tsconfig.json'],
		createDefaultProgram: true
	},
	parser: 'babel-eslint',
	plugins: ['react', 'prettier'],
	rules: {
		indent: ['error', 'tab'],
		quotes: ['error', 'single'],
		semi: ['error', 'never'],
		'react/react-in-jsx-scope': 'off',
		'react/prop-types': 'off',
		'no-var': 'error',
		'function-paren-newline': ['error', 'multiline'],
		'space-infix-ops': 'error',
		'array-bracket-spacing': ['error', 'never'],
		'object-curly-spacing': ['error', 'always'],
		'newline-per-chained-call': ['error', { ignoreChainWithDepth: 4 }],
		'object-property-newline': ['error', { allowAllPropertiesOnSameLine: true }],
		'linebreak-style': ['off', 'windows'],
		'jsx-quotes': ['error', 'prefer-single'],
		'comma-dangle': ['error', 'never'],
		'comma-spacing': 'error'
	},
	settings: {
		react: {
			version: 'detect'
		}
	}
}

```
run:
```
npm install qs -S
npm install @types/qs -D
```

warning:
```
WARNING in ./node_modules/side-channel/index.js
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):
Failed to load config "@ljharb" to extend from.
Referenced from: /Users/xx/workspace/node_modules/side-channel/.eslintrc
Error: Failed to load config "@ljharb" to extend from.
```

**Comment by ljharb** [comment]
This is because you're linting node_modules, which you shouldn't be. If you're in a monorepo with nested node_modules dirs, you have to add `**/node_modules` to your `.eslintignore`.

**Reply by JounQin** [reply]
![image](https://user-images.githubusercontent.com/8336744/121353936-f89cc880-c960-11eb-8a4e-10743ad7b15e.png)

@ljharb It doesn't help.

**Reply by ljharb** [reply]
If you pass a filename in node_modules directly to eslint, sure, but why would you ever do that? Run `eslint .`.

**Reply by JounQin** [reply]
vscode-eslint is doing this https://github.com/microsoft/vscode-eslint/issues/1240

I raised #14686 to track

**Reply by marcosdipaolo** [reply]
adding `**/node_modules` to `.eslintignore` file didn't work for me. Same error.

**Reply by ljharb** [reply]
@marcosdipaolo what file is eslint complaining about? is it inside node_modules?

**Reply by marcosdipaolo** [reply]
yes, same thing. It's a repo with the nested react app

**Reply by ljharb** [reply]
Try `**/node_modules/**`? there's no reason i'm aware of that shouldn't work.

**Reply by JounQin** [reply]
@marcosdipaolo This issue has not been not fixed, see also https://github.com/eslint/eslint/issues/14686#issuecomment-946270625

**Reply by chriscurnow** [reply]
> This is because you're linting node_modules, which you shouldn't be. If you're in a monorepo with nested node_modules dirs, you have to add `**/node_modules` to your `.eslintignore`.

Thanks so much for this. It took me a while to find this answer but of course it is spot on.

I was wondering why my linting was taking so long😀
